samples/Cordova/Backend/Startup.cs

```diff
@@ -80,9 +80,11 @@ public class Startup {
             app.UseOpenIdConnectServer(options => {
                 options.Provider = new AuthorizationProvider();
 
-                // Enable the authorization and logout endpoints.
+                // Enable the authorization, logout, token and userinfo endpoints.
                 options.AuthorizationEndpointPath = "/connect/authorize";
                 options.LogoutEndpointPath = "/connect/logout";
+                options.TokenEndpointPath = "/connect/token";
+                options.UserinfoEndpointPath = "/connect/userinfo";
```

Comment:
No, I used the token endpoint. Will comment the non needed endpoint to improve security.